gradle.properties

```diff
@@ -1,3 +1,4 @@
 # https://dev.to/jmfayard/configuring-gradle-with-gradle-properties-211k
 kotlin.code.style=official
-org.gradle.caching=true
\ No newline at end of file
+org.gradle.caching=true
+org.gradle.parallel=true
```

Comment:
Can you add an empty line at the end of the file to please git next times?

Reply:
I've added an editorconfig file, I think that does the trick automatically so that we can get that out of PR reviews